tests/python/frontend/mxnet/test_forward.py

```diff
@@ -1372,6 +1373,246 @@ def verify(data_shape, anchor_shape, stds=[1, 1, 1, 1], clip=-1, in_format="corn
+            for target, ctx in ctx_list():
+                for kind in ["graph", "vm", "debug"]:
+                    intrp = relay.create_executor(kind, mod=mod, ctx=ctx, target=target)
+                    op_res = intrp.evaluate()(data_np1, data_np2)
+                    tvm.testing.assert_allclose(op_res.asnumpy(), ref_res.asnumpy(), rtol=1e-5)
+    
+    verify(data_shape=(2,2))
+    verify(data_shape=(2,4))
+    verify(data_shape=(1,3,2))
+    verify(data_shape=(1,3,3))
+    verify(data_shape=(1,3))
+    verify(data_shape=(1,3,4))
+    verify(data_shape=(1,3,4))
+
+
+def test_forward_npi_binary_scalar():
+    def verify(data_shape, scalar):
+        ref_ops = [mx.np.power, mx.np.multiply, mx.np.add, mx.np.true_divide]
+        mx_ops = [mx.sym.np.power, mx.sym.np.multiply, mx.sym.np.add, mx.sym.np.true_divide]
+        for i in range(len(ref_ops)):
+            ref_op = ref_ops[i]
+            mx_op = mx_ops[i]
+            data_np1 = np.random.uniform(size=data_shape).astype("float32")
+            data1 = mx.sym.var('lhs')
+            ref_res = ref_op(mx.np.array(data_np1), scalar)
+            mx_sym = mx_op(data1.as_np_ndarray(), scalar)
+            mod, _ = relay.frontend.from_mxnet(mx_sym, shape={"lhs": data_shape}, dtype="float32")
+            for target, ctx in ctx_list():
+                for kind in ["graph", "vm", "debug"]:
+                    intrp = relay.create_executor(kind, mod=mod, ctx=ctx, target=target)
+                    op_res = intrp.evaluate()(data_np1)
+                    tvm.testing.assert_allclose(op_res.asnumpy(), ref_res.asnumpy(), rtol=1e-5)
+    
+    verify(data_shape=(2,2), scalar=1.0)
+    verify(data_shape=(2,4), scalar=2.0)
+    verify(data_shape=(1,3,2), scalar=3.0)
+    verify(data_shape=(1,3,3), scalar=4.0)
+
+
+def test_forward_npi_tanh():
+    def verify(data_shape):
+        data_np1 = np.random.uniform(size=data_shape).astype("float32")
+        data1 = mx.sym.var('data')
+        ref_res = mx.np.tanh(mx.np.array(data_np1))
+        mx_sym = mx.sym.np.tanh(data1.as_np_ndarray())
+        mod, _ = relay.frontend.from_mxnet(mx_sym, shape={"data": data_shape}, dtype="float32")
+        for target, ctx in ctx_list():
+            for kind in ["graph", "vm", "debug"]:
+                intrp = relay.create_executor(kind, mod=mod, ctx=ctx, target=target)
+                op_res = intrp.evaluate()(data_np1)
+                tvm.testing.assert_allclose(op_res.asnumpy(), ref_res.asnumpy(), rtol=1e-5)
+    
+    verify(data_shape=(2,2))
+    verify(data_shape=(2,4))
+    verify(data_shape=(1,3,2))
+    verify(data_shape=(1,3,3))
+
+
+def test_forward_npi_where_rscalar():
+    if not hasattr(mx.np, 'where'):
+        pytest.skip("mx.np.where hasn't been publish yet")
+
+    def verify(data_shape, scalar):
+        cond_np = np.random.uniform(size=data_shape).astype("bool")
+        data_np = np.random.uniform(size=data_shape).astype("float32")
```

Comment:
We may need to test for multiple dtypes. Especially for `cond_np`, we can have `bool` or `float32`.